tsdb_synth/README.md

```diff
@@ -0,0 +1,182 @@
+## TSDB k8s query track
+
+The main goal of the TSDB TSID HASH track is to measure the performance of TSID hashing both at index and query time.
+The dataset is synthetic and has been crafted to include many dimensions with different dimension name length and value size.
+
+### Generated data
+
+Generated data has the following characteristics:
+* Includes about 3.5 hours of data with about 256 time series per second. Per each epoch a complete set of all time series is generated with timestamps calculated to be in a small range of less than 1 second so to have data more distributed over time other then generating all time series at the same point in time.
+* Each time series is identified by a set of dimensions, one default `def_dim` dimension used as the `routing path` and 128 more dimensions whose names and values are
+  randomly generated. Names are generated with different lenghts while values are generated with lengths such that the total aggregated size of all dimensions per each
+  time series is (sometimes) larger then 32 Kb. The name of each dimension field starts with the prefix `d_` and ends with a progressive value as a suffix such as `001`, `002` and so on.
+* Other then the timestamp field and dimension fields, data includes two metric fields, a gauge `gauge` and a counter `counter` and two non-metric fields, `key` and `val`
+
+### Mapping properties
+
+```json
+{
+    "properties": {
+        "@timestamp": {
+            "type": "date"
+        },
+        "def_dim": {
+            "type": "keyword",
+            "time_series_dimension": "true"
+        },
+        "counter": {
+            "type": "long",
+            "time_series_metric": "counter"
+        },
+        "gauge": {
+            "type": "double",
+            "time_series_metric": "gauge",
+        },
+        "key": {
+            "type": "keyword"
+        },
+        "val": {
+            "type": "integer"
+        },
+        "d_ufseibmjnkguynqfyclevvweebkoxqgvspbbngkvtrsegqtwuyel_001": {"type": "keyword", "time_series_dimension": "true"},
```

Comment:
I will probbaly replace the "true" value here with a Jinja "statement" which uses "false" as default and that we can override by means of `track.params`.

Reply:
Or maybe I can change the maximum number of allowed dimensions.